conf/st2.dev.conf

```diff
@@ -11,6 +11,8 @@ mask_secrets = False
 
 [sensorcontainer]
 logging = st2reactor/conf/logging.sensorcontainer.conf
+sensor_node_name = sensornode1
+partition_provider = name:default
```

Comment:
It is a reference to the provider or name of the provider. I prefer default as `all` does not mean anything. Besides most often no one even needs to include this config as the value default is well default.